opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts

```diff
@@ -571,10 +573,16 @@ const attributePathMapping: any = {
     [ABSTRACT_STIX_RELATIONSHIP]: `/${objectLabel.stixName}`,
   },
   [INPUT_CREATED_BY]: {
-    [ABSTRACT_STIX_DOMAIN_OBJECT]: `/${createdBy.stixName}`,
+    [ABSTRACT_STIX_DOMAIN_OBJECT]: `/extensions/${STIX_EXT_OCTI}$/${createdBy.stixName}`,
```

Comment:
why this change ? created_by_ref doesn't seem to be part of octi extensions for stix domain objects